ts/packages/ts/src/commands/TestCommand.ts

```diff
@@ -0,0 +1,32 @@
+import { Command } from '../Command';
+import { ConfigStore } from '../ConfigStore';
+
+export type TestCommandParameters = {
+  store: ConfigStore;
+  clean?: boolean;
+};
+
+export class TestCommand extends Command<boolean> {
```

Comment:
see previous comment regarding void and error.

Reply:
done